### PR TITLE
Add 1 to `VTKPrimitives` connectivity indices to follow convention in ReadVTK.jl

### PR DESCRIPTION
**Pull request by eliascarv**
This convention is already working in the ".vtu" format and needs to be corrected in the ".vtp" format

### CODE REVIEW

File: src/ReadVTK.jl [hunk 1]
```diff
@@ -1069,7 +1069,8 @@ function get_cells(vtk_file::VTKFile)
   # Create VTKCells container and convert VTK's zero-based indices to Julia's one-based indices
   # Note: the offsets do not need to be updated since they point *past* the last entry
   # in VTK files (C-style), while in Julia it is custom to point *at* the last entry.
-  return VTKCells(get_data(connectivity) + oneunit.(get_data(connectivity)),
+  connectivity_data = get_data(connectivity)
+  return VTKCells(connectivity_data .+ oneunit(eltype(connectivity_data)),
```
Comment (sloede) [inline src/ReadVTK.jl:1073]: What's the effect of this change, i.e., why was it rewritten like this (honest question)?

**Review by sloede — COMMENTED** [review]
Thanks for the contribution! I have just one minor question.

@ranocha I think this will require a minor version bump since the observable behavior changes, what do you think?

File: src/ReadVTK.jl (hunk 1 above)
Comment (eliascarv) [reply src/ReadVTK.jl:1073]: The way it was being done before, an array of 1s was allocated to add to connectivity_data. 
I noticed that the `VTKDataArray{T}` will always contain elements of the same type.
Then I realized I could use broadcast to add `oneunit(T)` to all elements and save an allocation.

Comment (sloede) [reply src/ReadVTK.jl:1073]: Sounds reasonable 👍

**Review by sloede — APPROVED** [review]
LGTM, thanks!

**Review by ranocha — APPROVED** [review]
Thanks!

@sloede Feel free to go ahead as you proposed 👍

**Comment by sloede** [comment]
ReadVTK.jl v0.2.0 was just released, which includes this PR.

**Pull request merged**